xml_converter/src/xml_converter.cpp

```diff
@@ -104,9 +105,16 @@ void write_burrito_directory(
     }
     StringHierarchy category_filter;
     category_filter.add_path({}, true);
+    if (split_by_map_id && split_by_category_depth.has_value()) {
+        cerr << "Error: --split-by-category and --split-by-map-id can not be called at the same time" << endl;
```

Comment:
Theoretically this might not be a command line arg here. 

```suggestion
        cerr << "Error: 'split by category' and 'split by map id' can't be used at the same time" << endl;
```

Reply:
Added